ui/app/components/job-status/panel/steady.js

```diff
@@ -224,7 +224,7 @@ export default class JobStatusPanelSteadyComponent extends Component {
       };
     }
 
-    if (this.totalAllocs === 0) {
+    if (this.totalAllocs === 0 && !this.job.hasClientStatus) {
```

Comment:
single-job-index-page level status comes from this component computed property

Reply:
![image](https://github.com/user-attachments/assets/fdd1c5b1-62d5-4665-9280-7957e3008a23)